storage/log_storage.go

```diff
@@ -82,3 +83,11 @@ type LogRootWriter interface {
 	// StoreSignedLogRoot stores a freshly created SignedLogRoot.
 	StoreSignedLogRoot(root trillian.SignedLogRoot) error
 }
+
+// LogMetadata provides access to information about the logs in storage
+type LogMetadata interface {
+	// GetActiveLogs returns the IDs of all the logs that are configured in storage
+	// if filterPendingWorkOnly is true then the list of logs returned includes only those
+	// with pending leaves that need to be integrated into the log.
+	GetActiveLogIDs(filterPendingWorkOnly bool) ([]trillian.LogID, error)
```

Comment:
I wonder if this wouldn't just be better as two distinct API calls - they each seem to stand on their own as necessary functionality.